FWCore/Skeletons/python/cms.py

```diff
@@ -48,6 +48,17 @@ def config(tmpl, pkg_help, tmpl_dir):
     kwds['tmpl_etags'] = etags
     return kwds
 
+def config_with_parser(tmpl, args, tmpl_dir):
```

Comment:
please document the purpose of this function

Reply:
Sure, will add proper documentation.